dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -416,6 +416,21 @@ export const Card = ({
 			</Link>
 		);
 
+	const MediaPill = () => (
+		<div
+			css={css`
+				margin-top: auto;
+			`}
+		>
+			<Pill
+				prefix="Gallery"
+				content={galleryCount?.toString() ?? ''}
```

Comment:
If we want gallery count to be used as a string, would it be worth lifting this up and converting in the card enhancer?

Reply:
I think it makes sense that gallery count is a number, but was considering reworking the Pill component so that the conversion isn't needed. (Either extending the types that `content` accepts, or removing it entirely and making Pill a wrapper for child elements.)